example/lib/views/pubsub_view.dart

```diff
@@ -23,11 +24,12 @@ class PubSubController extends GetxController {
   final ion.Signal _signal = ion.GRPCWebSignal('http://localhost:9090');
   ion.Client _client;
   ion.LocalStream _localStream;
+  final String _uuid = Uuid().v4();
 
   void pubsub() async {
     if (_client == null) {
-      _client =
-          await ion.Client.create(sid: 'pub-sub-session', signal: _signal);
+      _client = await ion.Client.create(
+          sid: 'test room', uid: _uuid, signal: _signal);
```

Comment:
minor note: can we please use `test session` for the default sid so it is the same as `ion-sfu/examples/pubsubtest`? i know "test room" gets used in `ion-sdk-go` as well, i want to change all "test room" to "test session" so we are consistent

Reply:
Sure, This is reasonable.